core/src/avm1/globals/rectangle.rs

```diff
@@ -48,29 +48,29 @@ fn constructor<'gc>(
     args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
     if args.is_empty() {
-        this.set(istr!("x"), 0.into(), activation)?;
-        this.set(istr!("y"), 0.into(), activation)?;
-        this.set(istr!("width"), 0.into(), activation)?;
         this.set(istr!("height"), 0.into(), activation)?;
+        this.set(istr!("width"), 0.into(), activation)?;
+        this.set(istr!("y"), 0.into(), activation)?;
+        this.set(istr!("x"), 0.into(), activation)?;
     } else {
         this.set(
-            istr!("x"),
-            args.get(0).unwrap_or(&Value::Undefined).to_owned(),
+            istr!("height"),
+            args.get(3).unwrap_or(&Value::Undefined).to_owned(),
             activation,
         )?;
         this.set(
-            istr!("y"),
-            args.get(1).unwrap_or(&Value::Undefined).to_owned(),
+            istr!("width"),
+            args.get(2).unwrap_or(&Value::Undefined).to_owned(),
             activation,
         )?;
         this.set(
-            istr!("width"),
-            args.get(2).unwrap_or(&Value::Undefined).to_owned(),
+            istr!("y"),
+            args.get(1).unwrap_or(&Value::Undefined).to_owned(),
             activation,
         )?;
         this.set(
-            istr!("height"),
-            args.get(3).unwrap_or(&Value::Undefined).to_owned(),
+            istr!("x"),
+            args.get(0).unwrap_or(&Value::Undefined).to_owned(),
```

Comment:
Same as above, per the playerglobals.swf, non-empty rectangles are initialized in the "expected" order.